metanode/partition_store.go

```diff
@@ -670,9 +660,43 @@ func (mp *metaPartition) loadTxID(rootDir string) (err error) {
 	return
 }
 
-func (mp *metaPartition) loadUniqChecker(rootDir string) (err error) {
-	log.LogInfof("loadUniqChecker partition-%v begin", mp.config.PartitionId)
-	defer log.LogInfof("loadUniqChecker partition-%v complete", mp.config.PartitionId)
+func (mp *metaPartition) loadUniqID(rootDir string) (err error) {
+	filename := path.Join(rootDir, uniqIDFile)
+	if _, err = os.Stat(filename); err != nil {
+		err = nil
+		return
+	}
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		if err == os.ErrNotExist {
```

Comment:
here, notExist error should also return err.

Reply:
fixed